.github/ISSUE_TEMPLATE/bug-template.md

```diff
@@ -0,0 +1,25 @@
+Thank you for taking the time to let us know about the issue you found. Please provide us with the information below and we will look at it as soon as we are able.
+
+## Description
+
+*Please provide a short description of the bug*
+
+## Action Taken
+
+*In what way were you interacting with the application when you discovered the issue?*
+
+## What I expected to see
+
+*Please provide a short description of what you expected to see*
+
+## What I did see
+
+*Please provide a short description of what you did see. Screenshots are helpful, but please block out any personally identifying information before posting.*
+
+## Other Helpful Information
+
++ URL of the page I was on:
++ Browser and version:
++ Operating System:
++ Is the issue repeatable?: (yes | no | don't know)
```

Comment:
related question: has this bug occurred more than once when interacting with the app? (trying to distinguish between persistent and episodic bugs--though I'd defer to dev team on how much that experience matters for troubleshooting purposes 😄 ).

Reply:
I thought the repeatable question covered that, but I went ahead and added that also :)